keypair/keypair.go

```diff
@@ -0,0 +1,124 @@
+package keypair
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/mitchellh/go-homedir"
```

Comment:
You may consider using the standard library here: https://golang.org/pkg/os/#UserHomeDir

I don't think you need a third-party dependency for finding the home directory.

Reply:
I think I'll leave it as-is for now, but I appreciate the feedback! I think it's a good idea to rely less on third-party dependencies.